packages/calypso-products/src/get-jetpack-product-call-to-action.ts

```diff
@@ -12,15 +12,17 @@ import { getJetpackProductsCallToAction } from './translations';
 /**
  * Type dependencies
  */
-import type { Product } from './products-list';
+import type { Product } from './types';
 
 /**
  * Get Jetpack product call-to-action based on the product purchase object.
  *
  * @param {object} product Product purchase object
  * @returns {TranslateResult} Product display name
  */
-export function getJetpackProductCallToAction( product: object ): TranslateResult | undefined {
```

Comment:
`object` is not a friendly TS annotation and is better replaced by `Record< string, unknown >`.